src/App.js

```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react';
 import Header from '@/page/Header/Header';
 import MobileMenu from '@/components/MobileMenu/MobileMenu';
 import HeaderImage from '@/components/HeaderImage/HeaderImage'
+import AboutMe from './components/AboutMe/index'
```

Comment:
You can use short, it will be work
`import AboutMe from './components/AboutMe'`

Reply:
but better use name file how name component `AboutMe.js`

Reply:
use @ how alias